public/app/features/explore/TraceView/components/TracePageHeader/NewTracePageSearchBar.tsx

```diff
@@ -44,7 +45,9 @@ export default memo(function NewTracePageSearchBar(props: TracePageSearchBarProp
     totalSpans,
   } = props;
   const [currentSpanIndex, setCurrentSpanIndex] = useState(-1);
-  const styles = useStyles2(getStyles);
+  const theme = useTheme2();
+  const styles = getStyles(theme);
+  const buttonStyles = useStyles2(clearButtonStyles);
```

Comment:
Not sure I understand the styling here.

First I assume we can still use `const styles = useStyles2(getStyles);` can't we? Like it does not seem like we need the `theme` separately.

Are the `clearButtonStyles` unaccessible through the Button? I feel like it's weird that a style is directly exported from grafana/ui, never seen that pattern so wonder if that isn't something that may just go away at some point?